### DISCUSSION TITLE
Esp32-H2 support Zigbee 3.x

### DISCUSSION THREAD

**Discussion by osos** [Koenkk/zigbee2mqtt#8251] [Devices]
Have a look at this: https://www.espressif.com/en/news/ESP32_H2

Sounds very interesting.

**Comment by jasle** [comment]
The esp32-h2 is meanwhile available, including some modules with it. It's relatively cheap and has 20dBm output power, just like the cc2652p.

What would be needed to support the esp32-h2 in zigbee2mqtt?

**Comment by paede81** [comment]
Any updates? (Device is now widely available)
